paymentsheet-example/src/androidTest/java/com/stripe/android/test/core/PlaygroundTestDriver.kt

```diff
@@ -417,6 +419,43 @@ internal class PlaygroundTestDriver(
         return result
     }
 
+    fun confirmWithGooglePay(
```

Comment:
We will eventually want an `assumeTrue` or similar to make these not run unless we want them to (only for the new job).